Panels/Panels.module.less

```diff
@@ -82,17 +89,6 @@
 					font-family: @agate-base-font-family;
 				}
 			}
-
-			.viewport {
-				&.transitioning {
-					-webkit-clip-path: polygon(@agate-panels-breadcrumb-width 0, 100% 0, 100% 100%, @agate-panels-breadcrumb-width 100%);
-					clip-path: polygon(@agate-panels-breadcrumb-width 0, 100% 0, 100% 100%, @agate-panels-breadcrumb-width 100%);
-				}
-
-				> :not([data-index="0"]) {
-					width: calc(~"100% - " @agate-panels-breadcrumb-width);
-				}
-			}
```

Comment:
is this removal of styling for breadcrumbpanels needed?

Reply:
yes, because styling from here is not applied to the `Viewport` component